rtl/eclrtl/rtlfield.cpp

```diff
@@ -2639,6 +3172,16 @@ size32_t RtlIfBlockTypeInfo::size(const byte * self, const byte * selfrow) const
     return 0;
 }
 
+size32_t RtlIfBlockTypeInfo::buildString(ARowBuilder &builder, size32_t offset, const RtlFieldInfo *field, size32_t size, const char *value) const
+{
+    throwUnexpected();
```

Comment:
Some inconsistency about whether we are using rtlFailUnexpected() or throwUnexpected()